ChatIntegrationExamples/SendChatNotificationCard_1/SendChatNotificationCard_1.cs

```diff
@@ -0,0 +1,94 @@
+using AdaptiveCards;
+using Newtonsoft.Json;
+using Skyline.DataMiner.DcpChatIntegrationHelper.Common;
+using Skyline.DataMiner.DcpChatIntegrationHelper.Teams;
+
+namespace SendChatNotificationCard_1
+{
+	using System;
+	using System.Collections.Generic;
+	using Skyline.DataMiner.Automation;
+	
+	/// <summary>
+	/// Represents a DataMiner Automation script.
+	/// </summary>
+	public class Script
+	{
+		/// <summary>
+		/// The script entry point.
+		/// </summary>
+		/// <param name="engine">Link with SLAutomation process.</param>
+		public void Run(IEngine engine)
+		{
+			var chatIntegrationHelper = new ChatIntegrationHelperBuilder().Build();
+			try
+			{
+				var chatIdParam = engine.GetScriptParam("Chat Id");
+				if (string.IsNullOrWhiteSpace(chatIdParam?.Value))
+				{
+					engine.ExitFail("'Chat Id' parameter is required.");
+					return;
+				}
+
+				var notificationParam = engine.GetScriptParam("Notification");
+				if (string.IsNullOrWhiteSpace(notificationParam?.Value))
+				{
+					engine.ExitFail("'Notification' parameter is required.");
+					return;
+				}
+
+				var adaptiveCardBody = new List<AdaptiveElement>()
+				{
+					// The text notification from the input field
+					new AdaptiveTextBlock(notificationParam.Value)
+					{
+						Wrap = true
+					},
+					// Some additional examples
+					new AdaptiveFactSet()
+					{
+						Facts = new List<AdaptiveFact>
+						{
+							new AdaptiveFact("Name:", "Uplink"),
+							new AdaptiveFact("Bitrate:", "3 mb/s")
+						},
+					},
+					new AdaptiveFactSet()
+					{
+						Facts = new List<AdaptiveFact>
+						{
+							new AdaptiveFact("Name:", "Downlink"),
+							new AdaptiveFact("Bitrate:", "19 mb/s")
+						},
+					},
+					new AdaptiveImage("https://skyline.be/sites/default/files/inline-images/DataMinerbySLC_Q.png"),
+				};
+
+				try
+				{
+					chatIntegrationHelper.Teams.TrySendChatNotification(chatIdParam.Value, adaptiveCardBody);
+				}
+				catch (TeamsChatIntegrationException e)
+				{
+					engine.ExitFail($"Couldn't send the notification card to the chat with id {chatIdParam.Value} with error {e.Message}.");
```

Comment:
If this is text shown to the user, "id" should be written in capitals. 
```suggestion
					engine.ExitFail($"Couldn't send the notification card to the chat with ID {chatIdParam.Value} with error {e.Message}.");
```